### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by PetrushynskyiOleksii**
## JIRA

* [PYTHON-182](https://ssu-jira.softserveinc.com/browse/PYTHON-182)

## Code reviewers

- [ ] @meelros
- [ ] @BoroviyOrest
- [ ] @Kardan1123
- [ ] @mizin4ik
- [ ] @dimigor
- [x] @Sasha1152

### Second Level Review

- [ ] @lhalam

## Summary of change

- Update readme file

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,164 @@
...
+[![Build Status](https://travis-ci.org/Lv-365python/wayToHome.svg?branch=develop)](https://travis-ci.org/Lv-365python/wayToHome)
+[![Coverage Status](https://coveralls.io/repos/github/Lv-365python/wayToHome/badge.svg?branch=develop)](https://coveralls.io/github/Lv-365python/wayToHome?branch=develop)
+
+## Description
+WayToHome is web application for comfortable route tracking which allows setting up of
+notifications to get information about transport you want at desired time. The key features
+of system are as follows:
+- User can get optimal way from point A to point B.
+- User can get an account which allows him to set up notifications.
+- User has profile which contain google account photo, information about this user,
```
Comment (Kardan1123) [inline README.md]: which **contains**

File: README.md [hunk 2]
```diff
@@ -1 +1,164 @@
-# wayToHome
\ No newline at end of file
+# Way To Home
+[![Build Status](https://travis-ci.org/Lv-365python/wayToHome.svg?branch=develop)](https://travis-ci.org/Lv-365python/wayToHome)
+[![Coverage Status](https://coveralls.io/repos/github/Lv-365python/wayToHome/badge.svg?branch=develop)](https://coveralls.io/github/Lv-365python/wayToHome?branch=develop)
+
+## Description
+WayToHome is web application for comfortable route tracking which allows setting up of
```
Comment (Sasha1152) [inline README.md]: is **a** web application

File: README.md [hunk 3]
```diff
@@ -1 +1,164 @@
...
+notifications to get information about transport you want at desired time. The key features
+of system are as follows:
```
Comment (Sasha1152) [inline README.md]: of **the** system

**Review by MaxDubchak — APPROVED** [review]
Suits our project

**Review by dimigor — APPROVED** [review]

**Review by Sasha1152 — APPROVED** [review]

**Pull request merged**